acceptance-tests/tests/helpers/cmd.js

```diff
@@ -12,21 +12,31 @@ const PATH = process.env.PATH;
 
 /**
  * Creates a child process with script path
- * @param {string} processPath Path of the process to execute
+ * @param {string} cliPath Path of the CLI process to execute
+ * @param {string} cliNPMVersion NPM Version number
  * @param {Array} args Arguments to the command
  * @param {Object} env (optional) Environment variables
  */
-function createProcess(processPath, args = [], env = null) {
-  // Ensure that path exists
-  if (!processPath || !existsSync(processPath)) {
-    throw new Error('Invalid process path');
-  }
+function createProcess(cliPath, cliNPMVersion, args = [], env = null) {
+  let processCommand;
 
-  args = [processPath].concat(args);
+  if (cliNPMVersion) {
+    processCommand = 'npx';
```

Comment:
This will use npx to download and execute cli commands for each test. It ends up looking something like this:
```bash
npx --yes --package=@hubspot/cli@latest hs init
```